scripts/generate-counterpart-signature-pages.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+"""Generate framework agreement counterpart signature pages from supplier "about you" information for suppliers
+who applied to a framework.
+
+Currently will only work for framework_slug=g-cloud-8.
+
+To add support for future frameworks we will will need to add to the LOTS and DECLARATION_FIELDS in
+dmscripts/export_framework_applicant_details.py
+
+Usage:
+    scripts/generate-counterpart-signature-pages.py <stage> <api_token> <framework_slug> <template_folder> <out_folder>
+
+Example:
+    generate-counterpart-signature-pages.py dev myToken g-cloud-8 ../digitalmarketplace-agreements/documents/g-cloud pdf
+
+"""
+import sys
+import os
+import shutil
+
+sys.path.insert(0, '.')
+
+from docopt import docopt
+from dmscripts.env import get_api_endpoint_from_stage
+from dmscripts.export_framework_applicant_details import find_suppliers_with_details
+from dmscripts.generate_agreement_signature_pages import render_html_for_suppliers_awaiting_countersignature, \
+    render_pdf_for_each_html_page
+from dmapiclient import DataAPIClient
+from dmscripts import logging
+
+
+if __name__ == '__main__':
+    arguments = docopt(__doc__)
+
+    STAGE = arguments['<stage>']
+    API_TOKEN = arguments['<api_token>']
+    FRAMEWORK = arguments['<framework_slug>']
+    TEMPLATE_FOLDER = arguments['<template_folder>']
+    OUTPUT_FOLDER = arguments['<out_folder>']
+
+    repo_root = os.path.abspath(
+        os.path.join(os.path.dirname(__file__), '..'))
+    html_dir = os.path.join(OUTPUT_FOLDER, 'html')
+
+    os.makedirs(html_dir)
+
+    logger = logging.configure_logger()
+
+    client = DataAPIClient(get_api_endpoint_from_stage(STAGE), API_TOKEN)
+
+    headers, rows = find_suppliers_with_details(client, FRAMEWORK)
+
+    render_html_for_suppliers_awaiting_countersignature(rows, FRAMEWORK, TEMPLATE_FOLDER, html_dir)
+
+    html_pages = os.listdir(html_dir)
+    html_pages.remove('{}-signature-page.css'.format(FRAMEWORK))
+    html_pages.remove('{}-countersignature.png'.format(FRAMEWORK))
```

Comment:
This doesn't remove the files themselves (they need to be there for the rendering of PDFs to work), but removes their names from the list of files to be converted to PDFs (because trying to convert a css or png file to PDF will error).

I agree it's weird though.